### PR TITLE
Schema builder - group min height

### PR DESCRIPTION
**Pull request by ewels**
Set a minimum height for the content area of a group. This makes it easier to drop params into empty groups.

Very minor change, but makes it quite a bit easier to drop into new groups in my testing.

Before:

image

After:

image

Suggested by @ggabernet in nf-core/tools#626

### CODE REVIEW

File: public_html/assets/css/nf-core.css [hunk 1]
```diff
@@ -928,6 +928,7 @@ Based on https://codepen.io/wintr/pen/beBJBb */
 }
 .schema_group .card-body {
   padding: 1rem 1rem 0.5rem;
+  min-height: 50px;
```
Comment (mashehu) [inline public_html/assets/css/nf-core.css:931]:
```suggestion
  min-height: 1 rem;
```

Comment (ewels) [reply public_html/assets/css/nf-core.css:931]:
```suggestion
  min-height: 3rem;
```

**Comment by mashehu** [comment]
I am still not 100% happy how it looks like. How about some muted help text in there a la "drag and drop a parameter here"?

**Comment by ewels** [comment]
Ok, I got it to work with the helper text instead. I think it's nicer 👍 

Jun-24-2020 14-41-14

**Comment by ggabernet** [comment]
Even nicer like this!

**Comment by ewels** [comment]
@mashehu - happy with this now?

**Comment by mashehu** [comment]
yes! I would prefer if the text would disappear already while hovering with a grabbed parameter over the group, but one can't have everything 🙂

**Comment by ewels** [comment]
Yeah me too, but that was going to be more difficult 😅 Also having the text there actually makes it easier to drop..

**Pull request merged**